scrap/api.py

```
@@ -0,0 +1,51 @@
+from requests import get
+from dotenv import load_dotenv
+import os
+import math
+
+load_dotenv()
+PAGE_SIZE = 100
+MAX_START = 1000
+
+
+def fetch_api(query):
```

Ruff (0.11.9)

Missing return type annotation for public function `fetch_api`

(ANN201)